code/modules/cm_preds/falcon.dm

```diff
@@ -0,0 +1,108 @@
+	icon = 'icons/obj/items/hunter/pred_gear.dmi'
+	icon_state = "falcon_drone"
+	item_icons = list(
+		WEAR_L_EAR = 'icons/mob/humans/onmob/hunter/pred_gear.dmi',
+		WEAR_R_EAR = 'icons/mob/humans/onmob/hunter/pred_gear.dmi',
+		WEAR_L_HAND = 'icons/mob/humans/onmob/hunter/items_lefthand.dmi',
+		WEAR_R_HAND = 'icons/mob/humans/onmob/hunter/items_righthand.dmi'
+	)
+	flags_equip_slot = SLOT_EAR
+	flags_item = ITEM_PREDATOR
+
+/obj/item/falcon_drone/get_examine_location(var/mob/living/carbon/human/wearer, var/slot, var/t_he = "They", var/t_his = "their", var/t_him = "them", var/t_has = "have", var/t_is = "are")
+	switch(slot)
+		if(WEAR_L_EAR)
+			return "on [t_his] shoulder"
+		if(WEAR_R_EAR)
+			return "on [t_his] shoulder"
+	return ..()
+
+/obj/item/falcon_drone/verb/control_falcon_drone()
+	set name = "Control Falcon Drone"
+	set desc = "Activates your falcon drone."
+	set category = "Yautja.Misc"
+	set src in usr
+
+	if(usr.is_mob_incapacitated())
+		return
+
+	var/mob/living/carbon/human/H = usr
+	if(!istype(H) || !HAS_TRAIT(usr, TRAIT_YAUTJA_TECH))
+		to_chat(usr, SPAN_WARNING("You do not know how to use this."))
+		return
+
+	if(!istype(H.gloves, /obj/item/clothing/gloves/yautja))
+		to_chat(usr, SPAN_WARNING("You need your bracers to control \the [src]!"))
+		return
+
+	var/mob/hologram/falcon/hologram = new /mob/hologram/falcon(usr.loc, usr, src, H.gloves)
+	usr.drop_inv_item_to_loc(src, hologram)
+
+/mob/hologram/falcon
+	name = "falcon drone"
+	hud_possible = list(HUNTER_HUD)
+	var/obj/item/falcon_drone/parent_drone
+
+/mob/hologram/falcon/Initialize(mapload, mob/M, obj/item/falcon_drone/drone, obj/item/clothing/gloves/yautja/bracers)
+	. = ..()
+	parent_drone = drone
+	RegisterSignal(bracers, COMSIG_ITEM_DROPPED, .proc/handle_bracer_drop)
+	med_hud_set_status()
+	add_to_all_mob_huds()
+
+/mob/hologram/falcon/initialize_pass_flags(var/datum/pass_flags_container/PF)
+	..()
+	if(PF)
+		PF.flags_pass = PASS_MOB_THRU|PASS_MOB_IS
+		PF.flags_can_pass_all = PASS_ALL
+
+/mob/hologram/falcon/add_to_all_mob_huds()
+	var/datum/mob_hud/hud = huds[MOB_HUD_HUNTER]
+	hud.add_to_hud(src)
+
+/mob/hologram/falcon/remove_from_all_mob_huds()
+	var/datum/mob_hud/hud = huds[MOB_HUD_HUNTER]
+	hud.remove_from_hud(src)
+
+/mob/hologram/falcon/med_hud_set_status()
+	var/image/holder = hud_list[HUNTER_HUD]
+	holder.icon_state = "falcon_drone_active"
+
+/mob/hologram/falcon/Destroy()
+	if(parent_drone)
+		if(!linked_mob.equip_to_slot_if_possible(parent_drone, WEAR_L_EAR, TRUE, FALSE, TRUE, TRUE, FALSE))
+			if(!linked_mob.equip_to_slot_if_possible(parent_drone, WEAR_R_EAR, TRUE, FALSE, TRUE, TRUE, FALSE))
+				linked_mob.put_in_hands(parent_drone)
+		parent_drone = null
+	remove_from_all_mob_huds()
+	return ..()
+
+/mob/hologram/falcon/ex_act()
+	new /obj/item/trash/falcon_drone(loc)
+	QDEL_NULL(parent_drone)
+	qdel(src)
+
+/mob/hologram/falcon/emp_act()
+	new /obj/item/trash/falcon_drone/emp(loc)
+	QDEL_NULL(parent_drone)
+	qdel(src)
```

Comment:
can humans even see it? I couldn't during local testing

Reply:
nope, you can only see em with a pred mask on